apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -70,6 +66,7 @@ export const blocks = GoogleBlockly.common.createBlockDefinitionsFromJsonArray([
       'procedure_defnoreturn_set_comment_helper',
       'procedure_def_set_no_return_helper',
       'procedures_block_frame',
+      'procedure_def_mini_toolbox',
```

Comment:
I modified the name of this extension from `modal_function_mini_toolbox` to `procedure_def_mini_toolbox` because it applies to all procedures (including regular function definitions in the Dance Party edge case @mikeharv brought up today).